test/weighting/README

```diff
@@ -0,0 +1,17 @@
+
+Weighting Test-case
+===========================================
+
+This test uses the various aerosol weighting types on a single
+scenario to show that the results are the same between all the 
+different options. There are the following configurations:
+
+- run_part.spec: source class (n_class=3) with number and mass weighting (n_group=2)
+- run_part_flat_source.spec: source weighting (n_class=3) with number weighting (n_group=1)
+- run_part_flat.spec: single weight class (n_class=1) with number weighting (n_group=1)
+- run_part_numass.spec: single weight class (n_class=1) with number and mass weighting (n_group=2)
+- run_part_flat_specified: initial+background class and emission class (n_class=2)
+     with flat weighting (n_group=1)
+- run_part_nummass_specified: initial+background class and emission class (n_class=2)
+     with number and mass weighting (n_group=2)
+- run_exact.spec: exact solution
```

Comment:
Could we also add some gnuplot scripts to visualize the results of this new test case? It's hard to tell whether the tolerances have been set reasonably without plotting the data.

I'd also be ok with converting all the gnuplot scripts to python at some point, if that's better, although in this PR I think we should stick to gnuplot.

Reply:
Add two plot scripts - one for flat types and one for nummass types.